lib/api/core/models/repositories/userRepository.js

```diff
@@ -111,6 +112,13 @@ class UserRepository extends Repository {
     return super.load(id);
   }
 
+  async truncate (options) {
+    await super.truncate(options);
+
+    // @todo test unit
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)